src/Shopsys/ShopBundle/Model/Product/Pricing/ProductPriceCalculation.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+namespace Shopsys\ShopBundle\Model\Product\Pricing;
+
+use Shopsys\FrameworkBundle\Model\Pricing\BasePriceCalculation;
```

Comment:
Please change `FrameworkBundle` to `ShopBundle` in usages and annotations of classes that are extended in `ShopBundle`